tests/integration/Runner.php

```diff
@@ -81,6 +81,15 @@ private function getRunnableServices($service, $version)
         return $services;
     }
 
+    private function getCamelCaseStr($className) {
```

Comment:
all formatting should be consistent with [PSR-2](http://www.php-fig.org/psr/psr-2/). That means:

- 4 spaces for indents
- opening signature curly brace on separate line for classes and methods